### PR TITLE
feat: Show error on no selection

### LINKED ISSUE #15: Error on no selection
The command should return an error message when nothing is highlighted/selected.

The message should be `You have not highlighted anything`

### PR DESCRIPTION
**Pull request by coding-cosmos**
Fixes #15

### CODE REVIEW

**Comment by coding-cosmos** [comment]
@khattakdev Please check this.

File: src/extension.ts [hunk 1]
```diff
@@ -24,6 +24,11 @@ export function activate(context: vscode.ExtensionContext) {
         const selection = editor.selection;
         const text = document.getText(selection);
 
+        if (text.trim() === "") {
+          vscode.window.showErrorMessage("YOU HAVE NOT HIGHLIGHTED ANYTHING.");
```
Comment (khattakdev) [inline src/extension.ts:28]: Error message shouldn't be in caps.

Comment (coding-cosmos) [reply src/extension.ts]: Done. It's now "You have not highlighted anything."

**Comment by coding-cosmos** [comment]
@khattakdev  it's showing like this on hacktoberfest website can you check this out.
image

**Comment by khattakdev** [comment]
It should work now.

**Pull request merged**